### PR TITLE
feat: Career ad pages

### PR DESCRIPTION
**Pull request by sondrealf**
Closes: #146 

## Checklist

- [no] I have written tests
- [no] I have provided documentation

## Changelog

- Dark mode
- Career ad page 
- Works with sanity

## How to test

pnpm dev in web folder. Check sanity for example page (https://onlineweb-cms.sanity.studio/desk/career;66ad1d78-7af0-4f57-bf53-ef2e4b0486de)

### CODE REVIEW

File: apps/web/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "dev": "next dev",
     "build:prod": "next build",
     "start": "next start",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "start-storybook -p 6007",
```
Comment (matsjla) [inline apps/web/package.json:10]: Did this have to be changed?

File: apps/web/src/components/molecules/PortableText.tsx [hunk 2]
```diff
@@ -1,26 +1,21 @@
 /*eslint-disable*/
```
Comment (matsjla) [inline apps/web/src/components/molecules/PortableText.tsx:1]: Should probably not be here

File: apps/web/src/components/views/CareerAdView.tsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        <div className="bg-slate-12 mx-auto mt-10 mb-14 h-[0.5px] w-full" />
+
+        <div className={styles.infoTW}>
+          <BsArrowLeft size={20} className="text-blue-9 inline" />
+          <Text className="m-0 pl-2 text-xl font-bold text-blue-500"> ANDRE MULIGHETER</Text>
+        </div>
+
+        <div className="bg-slate-9 my-7 h-[0.5px] w-full" />
+
+        <div className={styles.infoTW}>
```
Comment (matsjla) [inline apps/web/src/components/views/CareerAdView.tsx:50]: Does this work with the styles? I think we should use tailwind classes instead

**Review by matsjla — CHANGES_REQUESTED** [review]
Looks good overall, just a few tiny nits and adjustments and we're good to go :rocket:

File: apps/web/package.json (hunk 1 above)
Comment (sondrealf) [reply apps/web/package.json:10]: To have two storybooks running the same time -yes. Both web and main storybook

File: apps/web/src/pages/career/[slug].tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+export const getServerSideProps: GetServerSideProps = async (ctx) => {
+  const slug = ctx.query.slug as string
+  const data = await fetchCareerAd(slug)
+  return { props: { career: data } }
+}
+
+const CareerAdPage: FC<CareerProps> = (props: CareerProps) => {
+  return props.career ? <CareerAdView career={props.career} /> : <div>404 - Sanity not found</div>
```
[the remark this answers is not in the document]
Comment (sondrealf) [reply apps/web/src/pages/career/[slug].tsx:17]: Dumdum bug

File: apps/web/src/components/molecules/PortableText.tsx (hunk 2 above)
Comment (sondrealf) [reply apps/web/src/components/molecules/PortableText.tsx:1]: It is supposed to be there since there is a "any" tag in Portable text

**Pull request merged**